libs/features/personalization/personalization.js

```diff
@@ -329,70 +332,84 @@ function registerInBlockActions(cmd, manifestId, targetManifestId) {
   config.mep.inBlock[blockName].commands.push(command);
 }
 
-function getSelectedElement(selector, action, rootEl) {
+function getSelectedElement({ selector, rootEl }) {
   if (!selector) return null;
-  if ((action.includes('appendtosection') || action.includes('prependtosection'))) {
-    if (!selector.includes('section')) return null;
-    const section = selector.trim().replace('section', '');
-    if (section !== '' && Number.isNaN(section)) return null;
-  }
```

Comment:
I don't think we can get rid of this completely.  This is trying to enforce that they don't prepend or append to anything but a section.  But I think we can improve this.  See below for suggestion.

Reply:
i missed this one. Good catch. Adding it back